spec/models/provider_interface/provider_applications_filter_spec.rb

```diff
@@ -67,6 +71,23 @@
           expect(headings).not_to include('Provider')
         end
       end
+
+      context 'when teacher degree apprenticeship feature flag active' do
+        let(:filter) do
+          described_class.new(params:,
+                              provider_user:,
+                              state_store:)
+        end
+
+        before do
+          FeatureFlag.activate(:teacher_degree_apprenticeship)
+        end
+
+        it 'does include the course type filter' do
+          expect(filter.filters.size).to be(7)
```

Comment:
Is this adding value? If we add another filter this will fail?

Reply:
i just maintaned the same test that was there and added the tda filter. I think I will maintain this.